dist/adaptive-html.es.js

```diff
@@ -380,10 +380,19 @@ rules.strong = {
     });
   }
 };
+rules.iframe = {
+  filter: 'iframe',
+  replacement: function replacement(content, node) {
+    var fallbackText = 'To view this content, please open this video in the Guru app';
```

Comment:
should this say something like `please open this iframe in the Guru app` vs video?

Reply:
fair question, i'll ask Andrew since he came up with the messaging

Reply:
yeah good call - gonna use a more generic message for non videos